### ISSUE TITLE
add tags description

### ISSUE THREAD

**Issue by pptime** [axnsan12/drf-yasg#273]
The possibility to describe tags like the following equivalent in yaml

tags:
- name: "pet"
  description: "Everything about your Pets"
  externalDocs:
    description: "Find out more"
    url: "http://swagger.io"
- name: "store"
  description: "Access to Petstore orders"
- name: "user"
  description: "Operations about user"
  externalDocs:
    description: "Find out more about our store"
    url: "http://swagger.io"

**Comment by axnsan12** [comment]
Tag definitions are added directly to the root ``Swagger`` object, which can be overriden by [OpenAPISchemaGenerator.get_schema](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#swagger-object).

**Comment by pptime** [comment]
Yes, this is reference of OpenAPI.  It might be better to have the full support of the standard.
